### ISSUE TITLE
Persian Translation

### ISSUE THREAD

**Issue by javadsaeid** [symfony/demo#1414]
Hi,
Symfony has a large community of Iranian developers and I suggest adding translation for the Persian language.

**Comment by OskarStark** [comment]
Feel free to open a PR 👍🏻

**Comment by javiereguiluz** [comment]
@javadsaeid yes, it'd be great to provide a Persian translation of this Demo application. Sadly none of the maintainers can speak that language, so please, provide a Pull Request or try to ask some people from the community to help you with this. We'll be happy to help you with the contribution process until this is merged. Thanks!

**Comment by javadsaeid** [comment]
@javiereguiluz Thank you for your response, sure ill send PR.

**Comment by setineos** [comment]
@javadsaeid any news ?
